src/app/shared/models/models.component.ts

```diff
@@ -509,11 +587,11 @@ export class ModelsComponent implements OnInit, OnDestroy {
     }
   };
 
-  classifierTreeOnSelect = node => {
+  classifierTreeOnSelect = (node) => {
```

Comment:
If in this area, might as well change this to `classifierTreeOnSelect(node) { }` instead